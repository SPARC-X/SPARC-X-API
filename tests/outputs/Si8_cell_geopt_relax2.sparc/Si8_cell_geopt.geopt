:RELAXSTEP: 1
:CELL:   9.3272727273E+00   9.3272727273E+00   9.3272727273E+00
:VOLUME:   8.1145422690E+02
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -5.6292942161E+01   8.8196137948E-04  -3.6637837366E-04 
  8.8196137948E-04  -5.6292757276E+01   3.2138320412E-04 
 -3.6637837366E-04   3.2138320412E-04  -5.6292624801E+01
:RELAXSTEP: 2
:CELL:   1.1286000000E+01   1.1286000000E+01   1.1286000000E+01
:VOLUME:   1.4375406617E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
  1.0020626895E+01  -5.9541735443E-10   4.2704367421E-09 
 -5.9541735443E-10   1.0020626902E+01  -5.7918792773E-10 
  4.2704367421E-09  -5.7918792773E-10   1.0020626897E+01
:RELAXSTEP: 3
:CELL:   1.1032774586E+01   1.1032774586E+01   1.1032774586E+01
:VOLUME:   1.3429326576E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
  7.1033863815E+00  -1.5545092313E-11  -6.8196461183E-10 
 -1.5545092313E-11   7.1033863823E+00  -2.8989733148E-11 
 -6.8196461183E-10  -2.8989733148E-11   7.1033863808E+00
:RELAXSTEP: 4
:CELL:   1.0441480472E+01   1.0441480472E+01   1.0441480472E+01
:VOLUME:   1.1383773384E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -5.1546636018E+00   4.1523866475E-13  -4.1382550152E-11 
  4.1523866475E-13  -5.1546636025E+00   6.6904849258E-12 
 -4.1382550152E-11   6.6904849258E-12  -5.1546636022E+00
:RELAXSTEP: 5
:CELL:   1.0698114243E+01   1.0698114243E+01   1.0698114243E+01
:VOLUME:   1.2243954131E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
  1.3144091941E+00  -8.5788897523E-10   1.3579211628E-08 
 -8.5788897523E-10   1.3144092015E+00  -5.5324285731E-10 
  1.3579211628E-08  -5.5324285731E-10   1.3144091977E+00
:RELAXSTEP: 6
:CELL:   1.0646967198E+01   1.0646967198E+01   1.0646967198E+01
:VOLUME:   1.2069179550E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
  1.8255733527E-01   1.1690610590E-10  -1.8251209346E-09 
  1.1690610590E-10   1.8255733453E-01   7.5006354546E-11 
 -1.8251209346E-09   7.5006354546E-11   1.8255733497E-01
:RELAXSTEP: 7
:CELL:   1.0638955458E+01   1.0638955458E+01   1.0638955458E+01
:VOLUME:   1.2041954223E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -1.4216711673E-03   1.0292458134E-10  -1.8760770611E-09 
  1.0292458134E-10  -1.4216727292E-03   7.2696789785E-11 
 -1.8760770611E-09   7.2696789785E-11  -1.4216721414E-03
:RELAXSTEP: 8
:CELL:   1.0639017414E+01   1.0639017414E+01   1.0639017414E+01
:VOLUME:   1.2042164603E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -5.6848955674E-06   4.1602791230E-12  -2.9734568924E-11 
  4.1602791230E-12  -5.6848342716E-06   1.2355227433E-12 
 -2.9734568924E-11   1.2355227433E-12  -5.6848234006E-06
