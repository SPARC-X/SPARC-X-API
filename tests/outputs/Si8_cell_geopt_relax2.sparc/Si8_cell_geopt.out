***************************************************************************
*                       SPARC (version Feb 03, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Sun Feb  5 13:18:38 2023                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
LATVEC_SCALE: 10.26 10.26 10.26 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 26 26 26
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001000000336
EXCHANGE_CORRELATION: GGA_PBE
NSTATES: 24
CHEB_DEGREE: 20
CHEFSI_BOUND_FLAG: 0
RELAX_FLAG: 2
CALC_STRESS: 1
MAXIT_SCF: 30
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 1.56E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_VELS: 1
PRINT_RESTART: 0
PRINT_ENERGY_DENSITY: 0
TOL_RELAX_CELL: 1.00E-04
RELAX_MAXDILAT: 1.10E+00
PRINT_RELAXOUT: 1
OUTPUT_FILE: Si8_cell_geopt/temp_run/Si8_cell_geopt
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors (Bohr):
10.260000000000000 0.000000000000000 0.000000000000000 
0.000000000000000 10.260000000000000 0.000000000000000 
0.000000000000000 0.000000000000000 10.260000000000000 
Volume: 1.0800455760E+03 (Bohr^3)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 24
NP_DOMAIN_PARAL: 1 1 2
NP_DOMAIN_PHI_PARAL: 3 4 4
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  48
Mesh spacing                       :  0.394615 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  Si8_cell_geopt/temp_run/Si8_cell_geopt.out
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  ../psps/14_Si_4_1.9_1.9_pbe_n_v1.0.psp8
Pseudocharge radii of atom type 1  :  7.50 7.50 7.50 (x, y, z dir)
Number of atoms of type 1          :  8
Estimated total memory usage       :  23.06 MB
Estimated memory per processor     :  492.04 kB
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 9.32727272727273 9.32727272727273 9.32727272727273 
CHEB_DEGREE: 20
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.358741 (Bohr)
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.1405228939E+00        2.679E-01        0.058
2            -4.1380545804E+00        1.955E-01        0.015
3            -4.1361562606E+00        1.494E-01        0.053
4            -4.1344485602E+00        3.835E-02        0.014
5            -4.1344391446E+00        3.094E-02        0.014
6            -4.1344472761E+00        2.303E-02        0.014
7            -4.1344696918E+00        1.498E-02        0.014
8            -4.1345082978E+00        1.502E-02        0.013
9            -4.1345323723E+00        1.734E-02        0.013
10           -4.1345600610E+00        2.518E-02        0.013
11           -4.1345772553E+00        3.086E-02        0.014
12           -4.1346164794E+00        4.247E-02        0.015
13           -4.1345434124E+00        1.408E-02        0.013
14           -4.1345367276E+00        7.442E-03        0.013
15           -4.1345357396E+00        5.380E-03        0.013
16           -4.1345348912E+00        1.621E-03        0.012
17           -4.1345349645E+00        7.551E-04        0.013
18           -4.1345350965E+00        6.470E-04        0.013
19           -4.1345351107E+00        4.480E-04        0.012
20           -4.1345351299E+00        3.016E-04        0.013
21           -4.1345351369E+00        1.562E-04        0.012
22           -4.1345351367E+00        1.366E-04        0.012
23           -4.1345351399E+00        9.894E-05        0.012
24           -4.1345351416E+00        7.923E-05        0.012
25           -4.1345351414E+00        4.882E-05        0.012
26           -4.1345351435E+00        3.420E-05        0.012
27           -4.1345351420E+00        2.403E-05        0.012
28           -4.1345351426E+00        2.016E-05        0.012
29           -4.1345351418E+00        8.315E-06        0.012
30           -4.1345351403E+00        7.547E-06        0.012
Total number of SCF: 30    
WARNING: SCF#1 did not converge to desired accuracy!
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -4.1345351403E+00 (Ha/atom)
Total free energy                  : -3.3076281122E+01 (Ha)
Band structure energy              :  1.7785366654E+00 (Ha)
Exchange correlation energy        : -1.2921501402E+01 (Ha)
Self and correction energy         : -5.7507565738E+01 (Ha)
-Entropy*kb*T                      : -1.0846846548E-02 (Ha)
Fermi level                        :  2.6123458948E-01 (Ha)
RMS force                          :  5.6771971647E-06 (Ha/Bohr)
Maximum force                      :  9.0163516394E-06 (Ha/Bohr)
Time for force calculation         :  0.029 (sec)
Pressure                           :  5.6292774746E+01 (GPa)
Maximum stress                     :  5.6292942161E+01 (GPa)
Time for stress calculation        :  0.055 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 11.286 11.286 11.286 
CHEB_DEGREE: 20
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.434077 (Bohr)
===================================================================
                    Self Consistent Field (SCF#2)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.2056251974E+00        3.333E-01        0.014
2            -4.1908216014E+00        2.575E-01        0.012
3            -4.1666168037E+00        5.001E-02        0.011
4            -4.1683738014E+00        3.391E-02        0.012
5            -4.1681705352E+00        8.056E-03        0.012
6            -4.1682154562E+00        3.073E-03        0.011
7            -4.1682138484E+00        3.474E-04        0.011
8            -4.1682139430E+00        8.434E-05        0.011
9            -4.1682139447E+00        2.950E-05        0.011
10           -4.1682139454E+00        6.951E-06        0.010
11           -4.1682139474E+00        5.891E-06        0.010
12           -4.1682139498E+00        4.959E-07        0.010
Total number of SCF: 12    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -4.1682139498E+00 (Ha/atom)
Total free energy                  : -3.3345711599E+01 (Ha)
Band structure energy              : -1.9442361296E+00 (Ha)
Exchange correlation energy        : -1.1822960499E+01 (Ha)
Self and correction energy         : -5.7507399472E+01 (Ha)
-Entropy*kb*T                      : -1.3286433214E-09 (Ha)
Fermi level                        :  1.0964918004E-01 (Ha)
RMS force                          :  1.8525632113E-10 (Ha/Bohr)
Maximum force                      :  3.6732267658E-10 (Ha/Bohr)
Time for force calculation         :  0.017 (sec)
Pressure                           : -1.0020626898E+01 (GPa)
Maximum stress                     :  1.0020626902E+01 (GPa)
Time for stress calculation        :  0.034 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 11.0327745859955 11.0327745859955 11.0327745859955 
CHEB_DEGREE: 20
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.424337 (Bohr)
===================================================================
                    Self Consistent Field (SCF#3)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.1720530083E+00        4.245E-02        0.013
2            -4.1719070600E+00        3.167E-02        0.011
3            -4.1716574728E+00        4.326E-03        0.011
4            -4.1716562812E+00        3.588E-03        0.011
5            -4.1716574077E+00        7.160E-04        0.011
6            -4.1716576100E+00        4.802E-04        0.011
7            -4.1716577539E+00        8.393E-05        0.011
8            -4.1716577593E+00        3.513E-05        0.011
9            -4.1716577571E+00        8.899E-06        0.010
10           -4.1716577602E+00        5.736E-06        0.010
11           -4.1716577630E+00        6.715E-07        0.010
Total number of SCF: 11    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -4.1716577630E+00 (Ha/atom)
Total free energy                  : -3.3373262104E+01 (Ha)
Band structure energy              : -1.5429878721E+00 (Ha)
Exchange correlation energy        : -1.1958636075E+01 (Ha)
Self and correction energy         : -5.7507457426E+01 (Ha)
-Entropy*kb*T                      : -5.0310582244E-09 (Ha)
Fermi level                        :  1.2609168579E-01 (Ha)
RMS force                          :  2.6136355405E-10 (Ha/Bohr)
Maximum force                      :  2.6161218655E-10 (Ha/Bohr)
Time for force calculation         :  0.014 (sec)
Pressure                           : -7.1033863815E+00 (GPa)
Maximum stress                     :  7.1033863823E+00 (GPa)
Time for stress calculation        :  0.027 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 10.4414804715436 10.4414804715436 10.4414804715436 
CHEB_DEGREE: 20
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.401595 (Bohr)
===================================================================
                    Self Consistent Field (SCF#4)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.1753519151E+00        9.783E-02        0.014
2            -4.1746338099E+00        7.210E-02        0.011
3            -4.1732644823E+00        1.044E-02        0.015
4            -4.1732463022E+00        7.706E-03        0.012
5            -4.1732533503E+00        1.443E-03        0.011
6            -4.1732546557E+00        9.016E-04        0.011
7            -4.1732554873E+00        1.677E-04        0.010
8            -4.1732554847E+00        3.378E-05        0.011
9            -4.1732554913E+00        1.092E-05        0.010
10           -4.1732554853E+00        1.989E-06        0.010
11           -4.1732554813E+00        1.295E-06        0.010
12           -4.1732554854E+00        2.750E-07        0.009
Total number of SCF: 12    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -4.1732554854E+00 (Ha/atom)
Total free energy                  : -3.3386043883E+01 (Ha)
Band structure energy              : -4.7748127959E-01 (Ha)
Exchange correlation energy        : -1.2312640612E+01 (Ha)
Self and correction energy         : -5.7507510906E+01 (Ha)
-Entropy*kb*T                      : -4.9415309422E-07 (Ha)
Fermi level                        :  1.6896329596E-01 (Ha)
RMS force                          :  8.6738951663E-12 (Ha/Bohr)
Maximum force                      :  9.4229193429E-12 (Ha/Bohr)
Time for force calculation         :  0.024 (sec)
Pressure                           :  5.1546636022E+00 (GPa)
Maximum stress                     :  5.1546636025E+00 (GPa)
Time for stress calculation        :  0.046 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 10.698114242681 10.698114242681 10.698114242681 
CHEB_DEGREE: 20
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.411466 (Bohr)
===================================================================
                    Self Consistent Field (SCF#5)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.1743310822E+00        4.115E-02        0.013
2            -4.1741683249E+00        3.067E-02        0.011
3            -4.1739087882E+00        4.498E-03        0.010
4            -4.1739099252E+00        3.634E-03        0.011
5            -4.1739161506E+00        7.967E-04        0.011
6            -4.1739166516E+00        4.171E-04        0.010
7            -4.1739168287E+00        6.594E-05        0.010
8            -4.1739168136E+00        2.256E-05        0.010
9            -4.1739168187E+00        5.995E-06        0.010
10           -4.1739168094E+00        2.573E-06        0.009
11           -4.1739168124E+00        4.905E-07        0.009
Total number of SCF: 11    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -4.1739168124E+00 (Ha/atom)
Total free energy                  : -3.3391334499E+01 (Ha)
Band structure energy              : -9.6439473023E-01 (Ha)
Exchange correlation energy        : -1.2152254535E+01 (Ha)
Self and correction energy         : -5.7507489070E+01 (Ha)
-Entropy*kb*T                      : -5.0570091164E-08 (Ha)
Fermi level                        :  1.4943233334E-01 (Ha)
RMS force                          :  5.3093402223E-11 (Ha/Bohr)
Maximum force                      :  5.3133726674E-11 (Ha/Bohr)
Time for force calculation         :  0.018 (sec)
Pressure                           : -1.3144091978E+00 (GPa)
Maximum stress                     :  1.3144092015E+00 (GPa)
Time for stress calculation        :  0.035 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 10.6469671983936 10.6469671983936 10.6469671983936 
CHEB_DEGREE: 20
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.409499 (Bohr)
===================================================================
                    Self Consistent Field (SCF#6)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.1739859365E+00        8.310E-03        0.013
2            -4.1739797157E+00        6.173E-03        0.011
3            -4.1739694836E+00        8.898E-04        0.010
4            -4.1739694601E+00        7.088E-04        0.010
5            -4.1739696191E+00        1.468E-04        0.010
6            -4.1739696402E+00        8.328E-05        0.010
7            -4.1739696386E+00        1.450E-05        0.010
8            -4.1739696381E+00        3.854E-06        0.009
9            -4.1739696363E+00        1.052E-06        0.010
10           -4.1739696329E+00        3.746E-07        0.009
Total number of SCF: 10    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -4.1739696329E+00 (Ha/atom)
Total free energy                  : -3.3391757063E+01 (Ha)
Band structure energy              : -8.7049562411E-01 (Ha)
Exchange correlation energy        : -1.2183386639E+01 (Ha)
Self and correction energy         : -5.7507490883E+01 (Ha)
-Entropy*kb*T                      : -7.6916002447E-08 (Ha)
Fermi level                        :  1.5321060671E-01 (Ha)
RMS force                          :  4.0471502708E-11 (Ha/Bohr)
Maximum force                      :  4.0497385303E-11 (Ha/Bohr)
Time for force calculation         :  0.023 (sec)
Pressure                           : -1.8255733492E-01 (GPa)
Maximum stress                     :  1.8255733527E-01 (GPa)
Time for stress calculation        :  0.045 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 10.6389554579841 10.6389554579841 10.6389554579841 
CHEB_DEGREE: 20
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.409191 (Bohr)
===================================================================
                    Self Consistent Field (SCF#7)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.1739704935E+00        1.298E-03        0.015
2            -4.1739703309E+00        9.645E-04        0.011
3            -4.1739700784E+00        1.396E-04        0.010
4            -4.1739700954E+00        1.113E-04        0.009
5            -4.1739700890E+00        2.324E-05        0.010
6            -4.1739700849E+00        1.297E-05        0.009
7            -4.1739700823E+00        2.235E-06        0.009
8            -4.1739700841E+00        6.133E-07        0.009
Total number of SCF: 8     
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -4.1739700841E+00 (Ha/atom)
Total free energy                  : -3.3391760673E+01 (Ha)
Band structure energy              : -8.5565119554E-01 (Ha)
Exchange correlation energy        : -1.2188299444E+01 (Ha)
Self and correction energy         : -5.7507492261E+01 (Ha)
-Entropy*kb*T                      : -8.2267809667E-08 (Ha)
Fermi level                        :  1.5380736821E-01 (Ha)
RMS force                          :  1.3483966392E-11 (Ha/Bohr)
Maximum force                      :  1.3520230786E-11 (Ha/Bohr)
Time for force calculation         :  0.023 (sec)
Pressure                           :  1.4216720126E-03 (GPa)
Maximum stress                     :  1.4216727292E-03 (GPa)
Time for stress calculation        :  0.045 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 10.6390174138585 10.6390174138585 10.6390174138585 
CHEB_DEGREE: 20
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.409193 (Bohr)
===================================================================
                    Self Consistent Field (SCF#8)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.1739700852E+00        1.000E-05        0.012
2            -4.1739700886E+00        7.441E-06        0.010
3            -4.1739700904E+00        1.095E-06        0.009
4            -4.1739700883E+00        8.802E-07        0.009
Total number of SCF: 4     
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -4.1739700883E+00 (Ha/atom)
Total free energy                  : -3.3391760706E+01 (Ha)
Band structure energy              : -8.5576742786E-01 (Ha)
Exchange correlation energy        : -1.2188260167E+01 (Ha)
Self and correction energy         : -5.7507492249E+01 (Ha)
-Entropy*kb*T                      : -8.2227125757E-08 (Ha)
Fermi level                        :  1.5380267088E-01 (Ha)
RMS force                          :  4.6790719258E-13 (Ha/Bohr)
Maximum force                      :  4.7860671582E-13 (Ha/Bohr)
Time for force calculation         :  0.023 (sec)
Pressure                           :  5.6848510799E-06 (GPa)
Maximum stress                     :  5.6848955674E-06 (GPa)
Time for stress calculation        :  0.045 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  2.237 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*               List of contributors: See the documentation               *
*         Citation: See README.md or the documentation for details        *
*  Acknowledgements: U.S. DOE SC (DE-SC0019410), U.S. DOE NNSA (ASC)      *
*      {Preliminary developments: U.S. NSF (1333500,1663244,1553212)}     *
***************************************************************************
                                                                           
